ckan/config/config_declaration.yaml

```
@@ -430,6 +430,12 @@ groups:
         description: >
           Defines how long (in seconds) requests calls should last before they will timeout.
 
+      - key: ckan.hide_version
+        default: False
+        example: True
+        type: string
```

This is probably causing the failures:
```suggestion
        type: bool
```